via/views/proxy.py

```diff
@@ -14,7 +14,9 @@ def static_fallback(_context, _request):
 @view_config(route_name="proxy", renderer="via:templates/proxy.html.jinja2")
 def proxy(context, request):
     url = context.url_from_path()
-    request.checkmate.raise_if_blocked(url)
+    checkmate_service = request.find_service(CheckmateService)
+
+    checkmate_service.raise_if_blocked(url)
```

Comment:
```suggestion
    request.find_service(CheckmateService).raise_if_blocked(url)
```

Doesn't look like we use this again?

Reply:
:+1: https://github.com/hypothesis/via/pull/1004/commits/b9feee694badad46fe59d6600da8bd33b91937ba